pkg/cache/tas_cache_test.go

```diff
@@ -221,8 +222,11 @@ func TestFindTopologyAssignment(t *testing.T) {
 		tolerations    []corev1.Toleration
 		wantAssignment *kueue.TopologyAssignment
 		wantReason     string
+		// TODO: remove after dropping the LeastAllocatedTAS feature gate
+		leastAllocated bool
```

Comment:
```suggestion
		enableLeastAllocated bool
```
nit, so that it is clear we are dealing with feature gate, and move it to the top.

Reply:
Done